Bottom: d080e0fdeac708f61640e985a278e64034e0ea77
Top:    7896699442a0a39dcb6816b79203e30abf7134f9
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-11 16:43:07 +0000

Refresh of work-in-progress-I

---

diff --git a/src/polygon1.c b/src/polygon1.c
index ad664cb..f793b0a 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -709,7 +709,7 @@ lookup_seg (PLINE * contour, VNODE * vertex)
   /* fill in the segment in info corresponding to this node */
   if (setjmp (info.sego) == 0)
     {
-      r_search_1 (contour->tree, &box, NULL, get_seg, &info);
+      r_search (contour->tree, &box, NULL, get_seg, &info);
       assert (0);
     }
   return info.s;
